### ISSUE TITLE
Not yet fully functional

### ISSUE THREAD

**Issue by WillRiches** [polaco1782/Agar.io-PHP-server#1]
As you have said, this server isn't fully functional. What needs to be done to make it functional? I might have some free time that I can commit to this project.
Thanks.

**Comment by WillRiches** [comment]
Actually spent more time looking at what you've got so far, didn't realise this is barebones at the moment! Do you have any idea how to actually format the data into the ArrayBuffer for the client to pick it up?

**Comment by polaco1782** [comment]
Yes. I'll commit more stuff soon. I reversed almost all communication format. I'll include with the test sources.
